test/functional/apps/dashboard/url_field_formatter.ts

```diff
@@ -74,14 +74,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await settings.controlChangeSave();
     });
 
-    it('applied on dashboard', async () => {
-      await common.navigateToApp('dashboard');
-      await dashboard.loadSavedDashboard('dashboard with everything');
-      await dashboard.waitForRenderComplete();
-      const fieldLink = await visChart.getFieldLinkInVisTable(`${fieldName}: Descending`, 1);
-      await clickFieldAndCheckUrl(fieldLink);
-    });
```

Comment:
is the table vis the only component that uses the url field formatter?

Reply:
For visualization, I see only Metrics and Table use url field formatter. But seems Metrics won't be affected. The url only shown in title. For metrics aggregations, url format has no impact.

<img width="724" alt="Screen Shot 2022-12-19 at 17 05 18" src="https://user-images.githubusercontent.com/79961084/208557834-eb456ad4-9701-4190-bddb-1352f9131e5d.png">

I see value from url format is now `"<span ng-non-bindable>10</span>"`. I personally think it is not necessary for Metrics to use url format. 

Will reproduce this on dashboard page in functional test repo.